frontstage/views/passwords/reset_password.py

```diff
@@ -96,7 +96,11 @@ def request_password_change(email):
     logger.info("Requesting password change", party_id=party_id)
 
     token = verification.generate_email_token(email)
-    verification_url = url_for('passwords_bp.post_reset_password', token=token)
+
+    # url_root comes with a trailing slash that we don't want.
+    url_root = request.url_root[:-1]
```

Comment:
is it a guarantee that it will always have a trailing slash? Is there any scenario where we might not get a trailing slash?

Reply:
Good question, there isn't a gaurantee (and the documentation in flask is actually wrong presently on this).  I'll put a check to not remove it if the last char isn't a slash

Reply:
Fixed